src/components/GlobalFilter/globalFilterApi.ts

```diff
@@ -5,7 +5,7 @@ import memoize from 'lodash/memoize';
 import { FlagTagsFilter, GroupItem } from '../../@types/types';
 import { getUrl } from '../../hooks/useBundle';
 
-export const SID_KEY = 'SAP ID (SID)';
+export const SID_KEY = 'SAP IDs (SID)';
```

Comment:
We should not be changing the output. Especially if the URL is in question. We will break bookmarks